### PR TITLE
Orca Yarn Test

### PR DESCRIPTION
**Pull request by sgwhat**

### CODE REVIEW

File: python/orca/example/learn/openvino/predict.py [hunk 1]
```diff
@@ -55,7 +55,7 @@ def crop(img, w, h):
 
     if args.cluster_mode == "local":
         init_orca_context(cores=args.core_num, memory=args.memory)
-    elif args.cluster_mode == "yarn":
+    elif args.cluster_mode.starswith("yarn"):
```
Comment (yushan111) [inline python/orca/example/learn/openvino/predict.py:58]: starts

**Review by yushan111 — COMMENTED** [review]
Shall we add orca integration tests to https://github.com/intel-analytics/BigDL/tree/branch-2.0/python/orca/dev/example

File: python/orca/example/learn/openvino/predict.py (hunk 1 above)
Comment (sgwhat) [reply python/orca/example/learn/openvino/predict.py:58]: Sure, it's better to put orca test scripts in orca/dev/example

**Comment by sgwhat** [comment]
> Shall we add orca integration tests to https://github.com/intel-analytics/BigDL/tree/branch-2.0/python/orca/dev/example

Sure, it's better to put orca test scripts in orca/dev/example

**Pull request closed without merging**